webpack.config.js

```diff
@@ -0,0 +1,245 @@
+const path = require('path');
```

Comment:
Since this is a SPA, please do the same I did for other SPAs with babel, webpack.*.js files

Reply:
Thank you. I updated it